lamindb_setup/core/upath.py

```
@@ -560,5 +561,6 @@ def check_s3_storage_location_empty(path: UPath) -> None:
         # regardless of the credentials registered in the path (see
         # lamindb_setup/core/_settings_storage/init_storage).
         raise ValueError(
```

Can you please raise a custom error?

```
class InstanceNotEmpty(Exception):
    pass
```